docs/resources.md

```diff
@@ -31,7 +31,6 @@ _Is this page missing a link? Open a PR!_
 ## Podcasts
 
 - [Code with Jason - ViewComponent with Joel Hawksley, Staff Engineer at GitHub](https://www.codewithjason.com/podcast/9936046-130-viewcomponent-with-joel-hawksley-staff-engineer-at-github/)
-- [Remote Ruby - ViewComponents and the Future of Assets](https://remoteruby.transistor.fm/125)
```

Comment:
I think the new URL for this podcast is https://www.remoteruby.com/2260490/13761017-viewcomponents-and-the-future-of-assets-with-joel-hawksley

Can you update it?

Reply:
Done. Thank you for the suggestions!